osquery/main/harnesses/gen_fuzz_config_dict.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# If this is not commented out, we get 'sed' commands in our output below
+#set -e
```

Comment:
Can you expand on that comment? I'm not sure why this would fail

Reply:
Sorry, you're right this is wrong. I removed it.